src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -0,0 +1,89 @@
+  ~       documentation and/or other materials provided with the distribution.
+  ~     * Neither the name of Salesforce.org nor the names of
+  ~       its contributors may be used to endorse or promote products derived
+  ~       from this software without specific prior written permission.
+  ~
+  ~     THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+  ~     "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+  ~     LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+  ~     FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+  ~     COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+  ~     INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+  ~     BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+  ~     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+  ~     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+  ~     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+  ~     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  ~     POSSIBILITY OF SUCH DAMAGE.
+  -->
+
+<template>
+    <div class="slds-m-bottom_medium">
+        <c-util-alert-banner theme="success"
+                             icon="utility:success"
+                             size="small"
+                             message="Gateway updated."
+                             is-dismissable="true"
+                             is-visible={isSuccess}>
+        </c-util-alert-banner>
+    </div>
+    <template if:true={isError}>
+        <div class="slds-m-bottom_medium">
+            <c-util-alert-banner theme="error"
+                                 icon="utility:error"
+                                 size="small"
+                                 message={errorMessage}
+                                 is-dismissable="true"
+                                 is-visible={isError}>
+            </c-util-alert-banner>
+        </div>
+    </template>
+    <lightning-layout multiple-rows="true">
+        <div if:true={showSpinner}
+             class='spinner-container'>
+            <lightning-spinner alternative-text={CUSTOM_LABELS.messageLoading}
+                               size='large'>
+            </lightning-spinner>
+            <p class='loading-text slds-text-heading_medium'>{loadingText}</p>
+        </div>
+        <lightning-layout-item size="12">
+            <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
+        </lightning-layout-item>
+        <lightning-layout-item size="12">
+            <p class="slds-text-body_regular slds-p-vertical_medium">NPSP uses the gateway that was added to Elevate most
+                recently. To override this, click Edit and enter the gateway ID for the one you'd like to use instead.
+                Find the gateway ID in the Elevate Admin Console | Gateway Management.</p>
```

Comment:
Reminder to leave inline comments about the hardcoded UI text.

Reply:
done!